### PR TITLE
feat(lock): add lock support

### PR DESCRIPTION
**Pull request by acesyde**
image

Requested feature : #313

### CODE REVIEW

File: src/cards/lock-card/controls/lock-buttons-control.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        this.hass.callService("lock", entry.serviceName!, {
+            entity_id: this.entity!.entity_id,
+        });
+    }
+
+    protected render(): TemplateResult {
+        const rtl = computeRTL(this.hass);
+
+        return html`
+            <mushroom-button-group .fill=${this.fill} .?tl=${rtl}
```
Comment (piitaya) [inline src/cards/lock-card/controls/lock-buttons-control.ts:56]:
```suggestion
            <mushroom-button-group .fill=${this.fill} .?rtl=${rtl}
```
Typo here 😅

File: src/cards/lock-card/controls/lock-buttons-control.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import { computeRTL, HomeAssistant } from "custom-card-helpers";
+import { html, LitElement, TemplateResult } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import { supportsFeature } from "../../../ha/common/entity/supports-feature";
+import { isAvailable } from "../../../ha/data/entity";
+import { LockEntity, LOCK_SUPPORT_OPEN } from "../../../ha/data/lock";
```
Comment (piitaya) [inline src/cards/lock-card/controls/lock-buttons-control.ts:6]: not used ?

File: src/cards/lock-card/controls/lock-buttons-control.ts (hunk 1 above)
Comment (acesyde) [reply src/cards/lock-card/controls/lock-buttons-control.ts:56]: You have the same bug in vacuum ^^

Comment (piitaya) [reply src/cards/lock-card/controls/lock-buttons-control.ts]: Oh ! We should fix that :p

**Comment by piitaya** [comment]
fix https://github.com/piitaya/lovelace-mushroom/issues/313

**Pull request merged**